db/migrate/20150204103835_add_state_to_defence_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToDefenceRequests < ActiveRecord::Migration
+  def change
+    add_column :defence_requests, :state, :string
```

Comment:
Do we want the initial state as the column default? I guess by leaving it as `NULL` in the database it only needs changing in the one place so I can see the argument for it - your call.